html_app/assignments/scb_ex/assignment.js

```diff
@@ -4906,7 +4906,7 @@ var __scb_sample_3 = {
                                 {
                                     name: 'Mouse anti-pEGFR',
                                     weight: 140,
-                                    intensity: 8,
```

Comment:
I get the switch above (not that I really understand it), but why the intensity change too?